pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -11,84 +11,19 @@
         "4.0.1.0": "4.0.1"
       }
     },
-    "Microsoft.NETCore": {
-      "StableVersions": [
-        "5.0.0",
-        "5.0.1",
-        "5.0.2"
-      ]
-    },
     "Microsoft.NETCore.Platforms": {
       "StableVersions": [
         "1.0.0",
         "1.0.1"
       ]
     },
-    "Microsoft.NETCore.Portable.Compatibility": {
```

Comment:
For WCF ones they are already defined in the WCF index.

None of these should ever be used by corefx, its at the bottom of the stack.  Its a problem (repo cycle) if we start depending on them.  I'd expect these to be defined in the repo where the packages live (coreclr, core-setup, or WCF).